Templates/CDS-CSharp/function.json

```diff
@@ -0,0 +1,16 @@
+﻿{
```

Comment:
@mattchenderson - You mentioned there is a setting that enables the EasyAuth warning on template function creation. What is that exactly, and where does that go?